jdaviz/core/launcher.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-  <div class="mx-12">
+  <div v-if="config_loaded">
+    <v-alert type="warning">
+      Jdaviz instance from this launcher has already loaded.  Create a new launcher instance or access launcher.jdaviz_helper.
```

Comment:
So only devs will see this warning?

Reply:
Anyone who uses the launcher from the notebook and then attempts to re-use that same instance would see this.  I don't expect this to be a common use-case outside of debugging, but you never know what people will try 🤷